### PR TITLE
fix(graph): add grayscale to technology icon

### PR DESCRIPTION
**Pull request by xiongemi**
## Related Issue(s)

Fixes #

### CODE REVIEW

**Comment by jaysoo** [comment]
Can we match the icons and colors to what's on Nx Cloud?
Screenshot 2024-05-02 at 8 16 08 AM

Icons: https://github.com/nrwl/ocean/tree/e9859b2206ab5b9e943862b13dc2cf488fad0f91/libs/nx-cloud/ui-icons/src/lib/technologies
Color: https://github.com/nrwl/ocean/blob/e9859b2206ab5b9e943862b13dc2cf488fad0f91/libs/nx-cloud/feature-runs/src/lib/tech-labels.tsx#L25

**Review by jaysoo — APPROVED** [review]

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]
The Gradle icon is broken

File: graph/ui-icons/src/lib/technology-icon.tsx [hunk 1]
```diff
@@ -1,33 +1,106 @@
...
+function UnknownTechnologyIcon({
+  technology,
+  showTooltip,
+}: TechnologyIconProps) {
+  if (!technology) {
+    return null;
+  }
+  return (
+    <div
+      className="flex h-4 w-4 items-center justify-center rounded bg-slate-800 text-sm text-slate-50 dark:bg-slate-50 dark:text-slate-800"
```
Comment (FrozenPandaz) [inline graph/ui-icons/src/lib/technology-icon.tsx:40]: While you're here, is it easy to make the icon in the project header (next to the Project Name) `h-6 w-6`? Right now it's really small next to the larger text.

image

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]

File: graph/ui-icons/src/lib/technology-icon.tsx (hunk 1 above)
Comment (xiongemi) [reply graph/ui-icons/src/lib/technology-icon.tsx]: changed 
Screenshot 2024-05-03 at 12 29 18 AM
Screenshot 2024-05-03 at 12 29 13 AM

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**